src/Skia/Avalonia.Skia.Desktop/RenderTarget.cs

```diff
@@ -127,9 +126,8 @@ public override DrawingContext CreateDrawingContext()
                 }
             }
 
-            var result =
-                new DrawingContext(
-                    new WindowDrawingContextImpl(this), Matrix.CreateScale(scale, scale));
+            // TODO: Broken DPI scaling on skia here. Fix this.
+            var result = new WindowDrawingContextImpl(this);
```

Comment:
I don't think that WindowRenderTarget (or RenderTarget in general) is used anymore. We should remove this file.